test/src/module/command/stanzaTest.c

```diff
@@ -512,6 +512,39 @@ testRun(void)
         TEST_RESULT_LOG(
             "P00   WARN: option --force is no longer supported\n"
             "P00   INFO: stanza-create for stanza 'db' on repo1");
+
+        // Clean-up file in archive directory
+        HRN_STORAGE_REMOVE(storageRepoIdxWrite(0), STORAGE_REPO_ARCHIVE "/somefile", .comment = "remove old test file");
+
+        // -------------------------------------------------------------------------------------------------------------------------
+        TEST_TITLE("stanza-create forcing another PG version");
+
+        argList = strLstDup(argListBase);
+        hrnCfgArgRawZ(argList, cfgOptPgVersion, "10");
```

Comment:
I think these tests would do better to write a v10 pg_control with a different control version. This tests reads a 9.6 pg_control file as 10, which seems to work, but is not what we are trying accomplish here.

Reply:
I initially thought it would be a good idea to check across PG version to see how it goes with extra fields in pg_control.
But you're right, it's best to keep it as clean as possible. Changed in ba1ba84 :-)